packages/app-desktop/gui/utils/NoteListUtils.ts

```diff
@@ -133,6 +134,18 @@ export default class NoteListUtils {
 				})
 			);
 
+			if (noteIds.length == 1) {
+				menu.append(
+					new MenuItem({
+						label: _('Copy note URL'),
```

Comment:
This is a bit confusing because we already have "Copy markdown link". I think we need to discuss all this in the spec and perhaps also see how it's done in other applications?

Reply:
Markdown link is a different format though.

I'm not sure how people would use this function but it seems logical to allow for easy creation of links to items that can be linked.

Reply:
How about calling this "Copy external link"? (and then same for tags and folders)